source/adapter.screenshare.js

```diff
@@ -6,7 +7,35 @@ AdapterJS.TEXT.EXTENSION = {
   BUTTON_CHROME: 'Go to Chrome Web Store'
 };
 
-AdapterJS.defineMediaSourcePolyfill = function () {
+// Define extension settings
+AdapterJS.extensionInfo =  AdapterJS.extensionInfo || {
+  chrome: {
+    extensionId: 'ljckddiekopnnjoeaiofddfhgnbdoafc',
+    extensionLink: 'https://chrome.google.com/webstore/detail/skylink-webrtc-tools/ljckddiekopnnjoeaiofddfhgnbdoafc',
+    // Flag for extension versions that does not honor the "mediaSource" flag
+    legacy: true,
+    // Deprecated! Define this to use iframe method
+    iframeLink: 'https://cdn.temasys.com.sg/skylink/extensions/detectRTC.html'
+  },
+  // Required only for Firefox 51 and below
+  firefox: {
+    extensionLink: 'https://addons.mozilla.org/en-US/firefox/addon/skylink-webrtc-tools/'
```

Comment:
@miniruwan so the case is that only Chrome needs the 2 flags because of the previously distributed extension that a bug that does not honor the `"mediaSource"` flag. The flag was maintained by default `true` to allow users to update the AdapterJS as a form of backwards compatibility without breaking their changes.

Firefox 52 onwards need not extensions to enable screensharing functionality, only for Firefox 51 and below.